docs/DeveloperGuide.md

```diff
@@ -260,6 +266,87 @@ Note that only the `Category` field is allowed to be specified multiple times, w
 a `ParserException` is thrown. Another noteworthy point is that `Category` that are added are to be case-insensitively unique and can only be up to 
 a specified value in the `UniqueCategoryList` class. Else, a `ParserException` would be thrown.
 
+#### Delete Transaction
+
+##### Overview
+
+The `DeleteCommand` function deletes an existing `Transaction` from `TransactionList` in UniCash.
+
+The activity diagram of deleting a Transaction is as shown below
+
+<img src="images/unicash/DeleteTransactionActivityDiagram.png" width="600" />
+
+The following sequence diagram shows the interaction between different components of UniCash.
+
+<img src="images/unicash/DeleteTransactionSequenceDiagram.png" width="1200" />
+
+The above sequence diagram omits details on the filtering of `TransactionList` and assumes that
+the displayed `TransactionList` is showing all transactions. However, the logic of the `DeleteCommand`
+remains the same for all list deletion.
+
+ℹ️ **Note:** The lifeline for `DeleteTransactionCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML,
```

Comment:
This is just a note for the following week. Do you think it might be better for this note to be added once at the top/bottom of the DG since it is going to be repeated for most of the commands entered?

Reply:
Yes that sounds like a good idea, can put maybe at the top or bottom and mention for all subsequent sequence diagrams

Reply:
Yeah I agree, we can specify general plant uml or markdown limitations at the top or at the the appendix